src/server/api/routers/webhook.ts

```diff
@@ -165,8 +172,10 @@ export const webhookRouter = createTRPCRouter({
         const response = await octokit.request(
           "DELETE /repos/{owner}/{repo}/hooks/{hook_id}",
           {
-            owner,
-            repo,
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, @typescript-eslint/no-non-null-asserted-optional-chain
+            owner: automation?.owner!,
```

Comment:
can't we gate this behind a conditional s.t. we only send the request of the automation exists? instead of always sending it and crashing if the automation doesn't exist?

Reply:
fixed! 😊